compiler/runtime/Runtime.hpp

```diff
@@ -159,6 +159,8 @@ enum TR_RuntimeHelper
    TR_numRuntimeHelpers = TR_ARM64numRuntimeHelpers
 #elif defined(TR_HOST_S390)
    TR_numRuntimeHelpers = TR_S390numRuntimeHelpers
+#elif defined(TR_HOST_RISCV)
+   TR_numRuntimeHelpers = TR_ARM64numRuntimeHelpers
```

Comment:
Indeed this is just a forgotten leftover from wild hacking. 

Rather than put literal 0, I tried to "the right thing" and follow the pattern, see commit [ceba23df](https://github.com/shingarov/omr/commit/ceba23dfaa4e4ace44b242682270058e8dcd8c50). 
All tests which passed before pass with this too. Let me know if you'd prefer "0" instead of my change, will change it.

Reply:
No, this is good and consistent with the other platforms.  Thanks for fixing.